src/Widgets/NotificationEntry.vala

```diff
@@ -47,6 +50,8 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
     }
 
     construct {
+        insert_action_group (ACTION_GROUP_PREFIX, NotificationMonitor.get_instance ().notifications_action_group);
```

Comment:
instead of inserting the action group in the entry, insert it in the listbox. child widgets can use action groups of parent widgets.

Reply:
done